### ISSUE TITLE
Market Transactions show wrong currency

### ISSUE THREAD

**Issue by KitsuneFM** [tfedor/AugmentedSteam#97]
![image](https://user-images.githubusercontent.com/47882260/53299476-164ece00-383b-11e9-9de1-ddb315b234e7.png)

The price shown in the left column is okay. But instead of my local currency, it shows the lowest price in euro, which is pretty unhelpful.

**Comment by MxtOUT** [comment]
Hm, can't replicate that.
Did you maybe set the currency under the price tab in the options page to EUR?
Can you have a look at your local storage [(instructions for Chrome)](https://developers.google.com/web/tools/chrome-devtools/manage-data/local-storage) and see what value `user_currency` has?

**Comment by KitsuneFM** [comment]
Yes, I set currency to EUR in the price tab. I always had this set to that on Enhanced Steam and it never made any issues. Store prices were in EUR, but Market prices were in PLN.
![image](https://user-images.githubusercontent.com/47882260/53303454-bcfe9300-386a-11e9-9fac-2fd95843d28b.png)

user_currency is set to PLN
![image](https://user-images.githubusercontent.com/47882260/53303422-685b1800-386a-11e9-86a8-f1c6bf8eacba.png)

Also the same problem appeared in Inventory page, when I wanted to sell the steam card few minutes ago
![image](https://user-images.githubusercontent.com/47882260/53303442-93456c00-386a-11e9-9170-8f466b3e21e9.png)

So basically I want things like market and inventory to show PLN, as it's my market region (unfortunatelly) but show lowest/historical prices of a games and apps in EUR. At least Store is like it is meant to be (PLN prices as I buy using that currency, but historical/lowest in EUR) ^^
![image](https://user-images.githubusercontent.com/47882260/53303504-3f875280-386b-11e9-9a56-3934689018cb.png)

**Comment by MxtOUT** [comment]
The originally described problem should be fixed with https://github.com/MxtOUT/Enhanced_Steam/commit/1698ecbad31fd5dc11fe717d3422d58022232015, I'll also have a look at the Quick Sell problem.

**Comment by KitsuneFM** [comment]
I should just update the community.js?

**Comment by MxtOUT** [comment]
Yeah, sure, right now only the market should work correctly.
As I said, I'll also have a look at the other problems :)

**Comment by KarlCastle** [comment]
> Also now comparing prices from different currencies in Store shows me EUR price in PLN currency. So It looks like that:
> ![image](https://user-images.githubusercontent.com/47882260/53380863-28ca2400-396f-11e9-80f1-9bf81a09b8cb.png)

That's a problem with the "eu1" region, and we're tracking it over in #76
